.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: publish
+
+on:
+  workflow_call:
+    inputs: 
+      source:
+        required: true
+        type: string
+    secrets:
+      apikey:
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the version
+        id: get_version
+        run: |
+          VERSION="${GITHUB_REF_NAME//v}"
+          echo VERSION:${VERSION}
+          echo ::set-output name=VERSION::${VERSION}  
+      - uses: actions/checkout@v2    
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+      - name: Install dependencies
+        run: dotnet restore -p:NuGetBuild=True
+      - name: Build
+        run: dotnet build --configuration Release --no-restore -p:NuGetBuild=True
+      - name: Test
+        run: dotnet test --no-restore -p:NuGetBuild=True
+      - name: Pack
+        run: dotnet pack --output artifacts --configuration Release --no-restore --no-build  -p:NuGetBuild=True -p:Version=${{ steps.get_version.outputs.VERSION }} -p:TreatWarningsAsErrors=false 
```

Comment:
Because when packing, referencing pre release packages is considered an error.
And internally OrchardCore is referencing Lucene (and others), which is technically still in beta, so it errors without this change, because this project references OrchardCore, which references Lucene etc...

Reply:
I'd rather turn that specific rule off here with `-p:TreatWarningsAsErrors=true`, with a comment on why we're doing it (so we can address any other `dotnet pack` warnings). We're fine using prerelease packages everywhere.

Reply:
It won’t pack if that warning is turned off. Unless the version number we specify is marked as a prerelease

Reply:
This creates the package for me correctly:

```
dotnet pack --output artifacts --configuration Release --no-restore --no-build  -p:NuGetBuild=True -p:Version="1.2.3" -p:NoWarn=NU5104 -p:TreatWarningsAsErrors=true 
```

Reply:
Ahh you meant turn the specific error code off. I misunderstood your request

Reply:
Done